tests/unit/modules/network/sonic/test_sonic_snmp.py

```diff
@@ -0,0 +1,143 @@
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+from ansible_collections.dellemc.enterprise_sonic.tests.unit.compat.mock import (
+    patch,
```

Comment:
I don't think these singular imports need to be enclosed in parentheses.

Reply:
Fixed.